src/data/sql-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const sqlQuiz = [
+  {
+    Question: 
+      "How can the user change “codingWebsite” into “FreeCodecamp” in the “Name” column in the Users table?",
+    Answer: "UPDATE Users Set Name = 'FreeCodecamp' WHERE Name = 'codingWebsite' ",
+    Distractor1: "MODIFY Users Set Name = 'codingWebsite' WHERE Name = 'FreeCodecamp' ",
+    Distractor2: "UPDATE Users Set Name = 'codingWebsite' WHERE Name = 'FreeCodecamp' ",
+    Distractor3: "MODIFY Users Set Name = 'FreeCodecamp' WHERE Name = 'codingWebsite' ",
```

Comment:
fixing incorrect spelling for freeCodeCamp.

```suggestion
    Answer: "UPDATE Users Set Name='freeCodeCamp' WHERE Name='codingWebsite' ",
    Distractor1: "MODIFY Users Set Name = 'codingWebsite' WHERE Name = 'freeCodeCamp' ",
    Distractor2: "UPDATE Users Set Name = 'codingWebsite' WHERE Name = 'freeCodeCamp' ",
    Distractor3: "MODIFY Users Set Name = 'freeCodeCamp' WHERE Name = 'codingWebsite' ",
```